internal/workflow/preprocessing.go

```diff
@@ -78,7 +78,7 @@ func (w *PreprocessingWorkflow) Execute(ctx temporalsdk_workflow.Context, params
 		return nil, e
 	}
 
-	// Repackage SFA Sip into a Bag.
+	// Repackage MOMA Sip into a Bag.
```

Comment:
It might be useful to have a the organization name as sips are not identical in structure but are actually 'typed' in that different orgs have different ways they structure what is submitted. In this case, moma does not as far as I am aware so I think it should be removed here.